pkg/agent/apiserver/apiserver.go

```diff
@@ -52,6 +53,7 @@ func (s *agentAPIServer) Run(stopCh <-chan struct{}) error {
 
 func installHandlers(aq monitor.AgentQuerier, npq monitor.AgentNetworkPolicyInfoQuerier, s *genericapiserver.GenericAPIServer) {
 	s.Handler.NonGoRestfulMux.HandleFunc("/agentinfo", agentinfo.HandleFunc(aq))
+	s.Handler.NonGoRestfulMux.HandleFunc("/getpodinterface", getpodinterface.HandleFunc(aq))
```

Comment:
Restful api doesn't have verb in url path, which should be expressed by http method.

Reply:
Updated.